kustomize/base/web-autoscaler.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: web
+spec:
+  scaleTargetRef:
+    apiVersion: v1
+    kind: DeploymentConfig
+    name: canvas-app-explorer-web-dev
+  minReplicas: 1
+  maxReplicas: 1 
```

Comment:
Do we want to bump the max?

Reply:
No, I think it's fine to leave this at just 1 based on the load tests for now.